frontend/src/component/admin/network/NetworkPrometheusAPIWarning.tsx

```diff
@@ -0,0 +1,24 @@
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+
+export const NetworkPrometheusAPIWarning = () => {
+    const {
+        uiConfig: { networkViewEnabled },
```

Comment:
We should probably rename `networkViewEnabled` to `prometheusAPIAvailable` or something.

I can do it in a follow-up PR if you agree, however it may be a breaking change according to our OpenAPI schema since it renames an existing property: https://github.com/Unleash/unleash/blob/14b6b38238fe5d9c8997600d33347bab539fa046/src/lib/openapi/spec/ui-config-schema.ts#L102

Reply:
Yes, but at the same time we discussed couple of times that our ui specific API should not be as strict as other APIs, so I don't mind too much

Reply:
Here you go: https://github.com/Unleash/unleash/pull/9330